### DISCUSSION TITLE
Wave header title color

### DISCUSSION THREAD

**Discussion by gbethardy-h2oai** [h2oai/wave#2155] [Q&A]
Is there a way to change the color of the header title independent of the theme?

**Comment by mturoci** [comment]
Hi @gbethardy-h2oai!

Header card supports `color` attribute with the following possible values: `'card' | 'transparent' | 'primary'`

Unfortunately, arbitrary colors are not supported. If you really want though, you can use [custom CSS](https://wave.h2o.ai/docs/custom-css/), but it's highly recommended to stick with the theme colors.
